src/structures/user.ts

```diff
@@ -37,6 +40,16 @@ export class User extends Base {
     return `<@${this.id}>`
   }
 
+  avatarURL(format: ImageFormats = 'webp', size: ImageSize = 512, dynamic: boolean = true): string {
+    return this.avatar != null
+      ? `${ImageURL(USER_AVATAR(this.id, dynamic ? this.avatar : this.avatar.replace('a_', '')), format, size)}`
+      : `${DEFAULT_USER_AVATAR(String(Number(this.discriminator) % 5))}.png`
```

Comment:
It always gets as png when the user doesn't have an avatar.
You should add format support to this too.

Reply:
![image](https://user-images.githubusercontent.com/61264156/100961132-1c6ff380-3565-11eb-9d4f-f76368f97d60.png)

default avatar no support size, format

Reply:
Oh yeah it does

![image](https://user-images.githubusercontent.com/10192625/100961318-7f618a80-3565-11eb-8621-fd3fb00fc651.png)

Reply:
And the size change is not supported in Default User Avatar either
![image](https://user-images.githubusercontent.com/67542663/100961695-e45ab780-3548-11eb-9b44-2455fced7e45.png)